docs/resources/iotda_access_credential.md

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "IoT Device Access (IoTDA)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_iotda_access_credential"
+description: -|
+  Manages an access credential resource within HuaweiCloud.
+---
+
+# huaweicloud_iotda_access_credential
+
+Manages an access credential resource within HuaweiCloud.
+
+-> 1.This resource is only a one-time action resource for doing API action. Deleting this resource will not clear
+  the corresponding request record, but will only remove the resource information from the tfstate file.
+  <br>2. If the API is called repeatedly, the access credential will be reset and the previous access credential
```

Comment:
```suggestion
  <br>2. The client will only retain one record. If the resource is redeployed, it will only reset the access credential, rendering the previous credential invalid.
```